pom.xml

```diff
@@ -35,14 +35,14 @@
     <cobertura.version>2.0.3</cobertura.version>
     <cobertura.maven.plugin.version>2.6</cobertura.maven.plugin.version>
     <maven.release.plugin.version>2.5.3</maven.release.plugin.version>
-    <spring-platform.version>2.0.8.RELEASE</spring-platform.version>
+    <spring-platform.version>Brussels-SR5</spring-platform.version>
```

Comment:
Really? They've moved from semantic versioning to names? Or is this some special pre-release versioning scheme they've come up with?

Reply:
Went and had a look at their site, so yes, they seem to have moved to "names" for these "meta" releases of their platform. I prefer SemVer but hey, nothing we can do about.

Reply:
Nothing we can do